docs/source/starters/starters.md

```diff
@@ -0,0 +1,81 @@
+```
+
+## Official Kedro starters
+
+The Kedro team maintains the following starters for a range of Kedro projects:
+
+* [`astro-airflow-iris`](https://github.com/kedro-org/kedro-starters/tree/main/astro-airflow-iris): An example project using the [Iris dataset](https://www.kaggle.com/uciml/iris) with a minimal setup for deploying the pipeline on Airflow with [Astronomer](https://www.astronomer.io/).
+* [`spaceflights-pandas`](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas): The [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) example code with `pandas` datasets.
+* [`spaceflights-pandas-viz`](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas-viz): The [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) example code with `pandas` datasets and visualisation and experiment tracking `kedro-viz` features.
+* [`spaceflights-pyspark`](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark): The [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) example code with `pyspark` datasets.
+* [`spaceflights-pyspark-viz`](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pyspark-viz): The [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) example code with `pyspark` datasets and visualisation and experiment tracking `kedro-viz` features.
+
+### Archived starters
+
+The following Kedro starters have been archived and are unavailable in Kedro version 0.19.0 and beyond.
+
+* [`standalone-datacatalog`](https://github.com/kedro-org/kedro-starters/tree/main/standalone-datacatalog)
+* [`pandas-iris`](https://github.com/kedro-org/kedro-starters/tree/main/pandas-iris)
+* [`pyspark-iris`](https://github.com/kedro-org/kedro-starters/tree/main/pyspark-iris)
+* [`pyspark`](https://github.com/kedro-org/kedro-starters/tree/main/pyspark)
+
+The latest version of Kedro that supports these starters is Kedro 0.18.14.
+
+* To check the version of Kedro you have installed, type `kedro -V` in your terminal window.
+* To install a specific version of Kedro, e.g. 0.18.14, type `pip install example-package<=0.18.14`.
+* To create a project with one of these starters using `kedro new`,  type the following (assuming Kedro version 0.18.14) `kedro new --starter=pandas-iris --checkout=0.18.14` (for example, to use the `pandas-iris` starter).
+
+
+## Starter versioning
+
+By default, Kedro will use the latest version available in the repository. If you want to use a specific version of a starter, you can pass a `--checkout` argument to the command:
+
+```bash
+kedro new --starter=spaceflights-pandas --checkout=0.1.0
+```
+
+The `--checkout` value can point to a branch, tag or commit in the starter repository.
+
+Under the hood, the value will be passed to the [`--checkout` flag in Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/usage.html#works-directly-with-git-and-hg-mercurial-repos-too).
+
+
+## Use a starter with a configuration file
```

Comment:
Happy to see this at the bottom of the page honestly, I think `--config` is hardly useful (especially now that we have `--name` at last)